spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/Invocation.java

```diff
@@ -34,7 +35,7 @@ class Invocation {
     JsonObject toJsonObject() {
         JsonObject result = new JsonObject();
         result.addProperty("exitCode", exitCode);
-        result.addProperty("exitSignalName", exitSignalName);
+        result.addProperty("exitSignalName", exitCodeDescription);
```

Comment:
@CodingDepot I'm confused here, why change part of the property name but leave it as exitSignalName?  I'm not seeing the purpose of the name change as a result here.  Its a bit confusing to me now.  Can you explain?

Reply:
No, you are completely right... I was being sloppy with the tacked on fix, I don't know how I missed that.
Upon review I found two more places where I failed to use the new property. :/
Thanks for having a closer look.